front/pages/api/w/[wId]/assistant/conversations/[cId]/content_fragment/index.ts

```diff
@@ -24,16 +24,7 @@ async function handler(
   >,
   auth: Authenticator
 ): Promise<void> {
-  const user = auth.user();
-  if (!user || !auth.isUser()) {
-    return apiError(req, res, {
-      status_code: 404,
-      api_error: {
-        type: "workspace_user_not_found",
-        message: "Could not find the user of the current session.",
-      },
-    });
-  }
+  const user = auth.getNonNullableUser();
```

Comment:
No this can't throw because the wrapper `allowUserOutsideCurrentWorkspace` checks for the non-nullity of owner and return a proper error otherwise :+1: